Framework/Algorithms/test/MCInteractionVolumeTest.h

```diff
@@ -167,6 +170,97 @@ class MCInteractionVolumeTest : public CxxTest::TestSuite {
                                                     lambdaBefore, lambdaAfter),
                      const std::runtime_error &);
   }
+
+  void testGeneratePointConsidersAllComponents() {
+    using namespace ::testing;
+    using namespace Mantid::Kernel;
+    using namespace MonteCarloTesting;
+
+    auto kit = createTestKit();
+    size_t maxAttempts(1);
+
+    using Mantid::API::Sample;
```

Comment:
These using statements can be hoisted out of each test into the top of the test file

Reply:
OK done